[Doc enhancement][Addnote][1815255] (#3535)
MicrosoftDocs/msteams-docs: msteams-platform/concepts/design/enable-app-customization.md

---
# Enable orgs to customize your Microsoft Teams app

You can allow customers to customize some aspects of your Microsoft Teams app in the Teams admin center. This feature is supported only for apps published to the Teams store. Sideloaded apps and apps published for an org can't be customized.

Some possible examples of this feature include:

* Changing the app's accent color to match an org's brand.
* Updating the app name from *Contoso* to *Contoso Agent*, which is the name users in the org will see. (Note: Users adding a connector to a chat or a channel will still see the original app name, *Contoso*.)

> You can enable this feature in the [Developer Portal for Teams](https://dev.teams.microsoft.com/home). This configures `configurableProperties` that are not available in previous versions prior to 1.10 of the Teams app manifest.

Reviewer: ```suggestion
You can enable this feature in the [Developer Portal for Teams](https://dev.teams.microsoft.com/home). This configures `configurableProperties`, which aren't available in versions prior to 1.10 of the Teams app manifest.
```

## Test your app

> To test app customization, submit the latest version of app manifest, and use a test tenant to verify your changes. App customization does not support sideloading. You cannot test the feature by sideloading the app. 

Reviewer 2: "App customization is not supported in sideloaded apps" would be more appropriate

Reviewer: @vganiga Is it OK if we remove this section? Based on our discussions, this info is incorporated elsewhere in this doc and the related admin doc (where the info is probably more relevant). The "Test your app" heading also implies developer testing when it's really more about the customer admin doing the testing.

Reviewer 2: @heath-hamilton Although they dont have to test anything, developers are used to sideloading and testing their app before sending for validation, so its better to add this note so they know why something isnt as they expected.

Reviewer: @vganiga That makes sense. Would something like this work instead where we definitively tell the dev they can't test during development?

## Test your app

You can't test this feature during development. App customization isn't supported for apps sideloaded or published to an org's app catalog.

Reviewer 2: yeah that works too

Reviewer: @Mahesh2516, please update the text to reflect this:

## Test your app

You can't test this feature during development. App customization isn't supported for apps sideloaded or published to an org's app catalog.

Author: Updated

You can allow orgs (specifically Teams admins) to customize some aspects of your Microsoft Teams app. Some possible examples include:

* Changing the app's accent color to match the org's brand.

> * Updating the app name from *Contoso* to *Contoso Agent*, which is the name users in the org will see. 
>    > [!NOTE] 
>    > Users adding a connector to a chat will still see the original app name, *Contoso*.

Reviewer: ```suggestion
* Updating the app name from *Contoso* to *Contoso Agent*, which is the name users in the org will see.  (Note: Users adding a connector to a chat will still see the original app name, *Contoso*.)
```
Suggest reverting this to an inline note to reduce visual noise. In general, note stylings should be reserved for things that affect the entire feature. I am not sure this does. Also, does this apply to adding a connector to a channel?

You can enable this feature in the [Developer Portal for Teams](https://dev.teams.microsoft.com/home). (This configures `configurableProperties` in your app manifest.) For more information, see [customize apps in Microsoft Teams](/MicrosoftTeams/customize-apps).

> [!NOTE]

> > App customization is not supported for sideloaded and LOB apps. You cannot test the feature by sideloading the app.

Reviewer: Does this mean this feature is only available for apps published to the Teams store? If so, we should probably state that instead. We could still keep the second sentence about sideload testing.

Reviewer 2: I agree to Heath's comment here, also instead of adding it as a note, can we mention this as Testing instructions? 
`Testing Instructions: 
App customization is not supported for sideloaded and LOB apps. You cannot test the feature by sideloading the app. To test this feature, submit the new app manifest and use a test tenant to verify your changes`

Reviewer 3: Edit as per suggestion.
